Modules/Core/Common/include/itkLoggerThreadWrapper.h

```diff
@@ -136,6 +150,10 @@ class ITK_TEMPLATE_EXPORT LoggerThreadWrapper:public SimpleLoggerType
   DelayType m_Delay;
 
 };  // class LoggerThreadWrapper
+
+// Define how to print enumeration
+extern ITKCommon_EXPORT std::ostream& operator<<(std::ostream& out, const LoggerThreadWrapperOperationType value);
```

Comment:
In SimpleITK procedural methods I din't need the "extern" specifier.

Reply:
@blowekamp Thanks.  I think it is implied by default if it is not there.  I prefer explicitly declaring that this procedural method is extern.